modules/install/pages/migration.adoc

```diff
@@ -1,80 +1,93 @@
+
+. After the data restore operation is complete on Cluster 1, delete Bucket A’ from Cluster 2.
+
+. Create Bucket B’ on Cluster 2 and select the Timestamp-based Conflict Resolution type.
+
 . Create a new unidirectional replication for Bucket B.
 See xref:manage:manage-xdcr/xdcr-management-overview.adoc[XDCR Management Overview].
 
-For Scenario 2, perform the following steps to migrate your data to a new cluster:
+== Migration during Bidirectional Replication
+
+Follow these steps to migrate your data to a new bucket during bidirectional replication:
 
-. Stop application traffic coming to Cluster 1 (Bucket A) and Cluster 2 (Bucket A’).
+. Stop application traffic coming into Cluster 1 (Bucket A) and Cluster 2 (Bucket A’).
 +
-NOTE: Note: Allow _enough time_ for the replication queues to drain.
-Check the *outbound XDCR mutation* statistics to confirm.
-For instructions, see
-xref:manage:monitor/ui-monitoring-statistics.adoc#outgoing_xdcr_stats[Monitoring Outgoing XDCR].
+NOTE: Allow enough time for the replication queues to drain.
+Confirm by reviewing the XDCR Total Outbound Mutations statistic.
+For more information, see xref:manage:monitor/monitor-intro.html#monitoring-with-the-ui[].
 
-. Stop and delete replication to both clusters.
+. Stop and delete the replication streams to both clusters.
 For instructions, see
 xref:manage:manage-xdcr/xdcr-management-overview.adoc[XDCR Management Overview].
-. Run the `cbbackupmgr` tool to backup the entire bucket data on both clusters.
+
+. Run the `cbbackupmgr` tool on both clusters to backup the bucket data.
 For instructions, see
 xref:backup-restore:enterprise-backup-restore.adoc[Backup].
+
 . Delete the Bucket A from Cluster 1 and Bucket A’ from Cluster 2.
 For instructions, see
 xref:manage:manage-buckets/delete-bucket.adoc[Delete a Bucket].
-. Upgrade both clusters to Couchbase Server version 4.6.0.
-For instructions, see xref:upgrade.adoc[Upgrading Couchbase Server].
-. Create new buckets on both clusters with the conflict resolution type as *Timestamp* selected.
+
+. Create buckets on both clusters and select the Timestamp-based Conflict Resolution type.
 For instructions, see xref:manage:manage-buckets/create-bucket.adoc[Create a Bucket].
+
 . Run the `cbbackupmgr` tool to restore data.
-When restoring data from backup (use the [.cmd]`--force-updates` option).
-Make sure to disable *Conflict Resolution* option during the restore.
-This is required because the conflict resolution types of the source and destination do not match.
-. Once the restore operation is completed on both clusters, create replication streams both ways from Cluster 1 and Cluster 2.
-See
-xref:manage:manage-xdcr/xdcr-management-overview.adoc[XDCR Management Overview].
+Turn off the Conflict Resolution setting on the destination cluster and then use the `--force-updates` option to restore data from backup.
+Do this because the source and destination clusters have different conflict resolution types.
```

Comment:
This is for lines 89 and 90.

Use the cbbackupmgr restore option `--force-updates` to restore data from the backup.  This option disables conflict resolution during the restore, and it is needed because the bucket you are restoring to has a different conflict resolution type than the bucket that the backup is from.  Note that the `--force-updates` option also updates the CAS of the documents being restored.